package.json

```diff
@@ -7,19 +7,25 @@
     "node": ">=7.6"
   },
   "scripts": {
+    "prepublish": "npm run build",
     "lint": "standard --fix",
-    "test": "node --harmony node_modules/.bin/ava",
-    "test:watch": "node --harmony node_modules/.bin/ava --watch",
-    "test:coverage": "nyc node --harmony node_modules/.bin/ava",
+    "test": "npm run build; node --harmony node_modules/.bin/ava",
```

Comment:
@bluzi we should use `&&` between the commands instead of `;` so that the 2nd part is only run if the first part is successful. can you make the change across the other commands as well? (test:watch, etc)